### PR TITLE
doc (davide): add Evogym english documentation

### PR DESCRIPTION
**Pull request by davide97l**

### CODE REVIEW

File: source/13_envs/evogym.rst [hunk 1]
```diff
@@ -0,0 +1,226 @@
+Evogym
+~~~~~~~~~
+
+Overview
+=========
+
+Evolution Gym (Evogym) is the first large-scale benchmark for co-optimizing the design and control of soft robots.
+Each robot is composed of different types of voxels (e.g., soft, rigid, actuators), resulting in a modular and expressive robot design space.
+The environment spans a wide range of tasks, including locomotion on various types of terrains and manipulation.
```
Comment (zjowowen) [inline source/13_envs/evogym.rst:9]: The environment spans a wide range of tasks, including locomotion and manipulation on various types of terrains.

File: source/13_envs/evogym.rst [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+- The observation space includes the state information of the robot, a\ ``(2N + 3)``\ vector including the relative position of each voxel corner with respect to the center of mass of the robot \ ``(2N)``\, and the velocity and orientation of center of mass \ ``(3)``\. The data type is\ ``float32``\.
+- To handle complex tasks, specifically those with varying terrain types, an additional observation vector including terrain information is provided.
+- Furthermore, goal-related information is offered to inform the controller of the execution status of the current task.
+- For instance, in manipulation tasks where the robot interacts with some object O, it is provided orientation and velocity as well as the position of O’s center of mass relative to the robot.
+
+Action Space
+-------------
+
+- It is a a continuous action space of size N. Each of the N components of the action vector is associated with an actuator voxel (either horizontal or vertical) of the robot, and instructs a deformation target of that voxel. The data type is \ ``float``\.
```
Comment (zjowowen) [inline source/13_envs/evogym.rst:54]: The action space is continuous of size N. Each component of the N-size action vector is associated with an actuator voxel (either horizontal or vertical) of the robot, and instructs a deformation target of that voxel, compressing or stretching the size of voxel from 60% to 160%。

File: source/13_envs/evogym.rst [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+- The training environment uses a dynamic random seed, that is, the random seed of each episode is different, and is generated by a random number generator, but the seed of this random number generator is fixed by the \ ``seed`` \ method of the environment ;The test environment uses a static random seed, that is, the random seed of each episode is the same, specified by the \ ``seed`` \ method.
+
+Store Video
+------------
+
+After the environment is created, but before reset, call the \ ``enable_save_replay`` \ method, specifying the path to save the game replay. The environment will automatically save the local video files after each episode ends. (The default call \ ``gym.wrappers.RecordVideo`` \ implementation ), the code shown below will run an environment episode and save the result of this episode in a folder\ ``./video/``\ :
+
+.. code:: python
+
+    import gym
```
Comment (zjowowen) [inline source/13_envs/evogym.rst:112]: replace with:

import time
import gym
from evogym import sample_robot
# import envs from the envs folder and register them
import evogym.envs

if __name__ == '__main__':
    # create a random robot
    body, connections = sample_robot((5, 5))
    env = gym.make('Walker-v0', body=body)

    if gym.version.VERSION > '0.22.0':
        env.metadata.update({'render_modes': ["rgb_array"]})
    else:
        env.metadata.update({'render.modes': ["rgb_array"]})
    env = gym.wrappers.RecordVideo(
        env,
        video_folder="./video",
        episode_trigger=lambda episode_id: True,
        name_prefix='rl-video-{}'.format(time.time())
    )

    env.reset()
    # step the environment for 100 iterations
    for i in range(100):
        action = env.action_space.sample()
        ob, reward, done, info = env.step(action)
        x = env.render()
        if done:
            env.reset()
    env.close()

File: source/13_envs/evogym.rst [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+.. image:: ./images/evogym.gif
+   :align: center
+
+Install
+========
+
+Installation Method
+--------------------
+
+The environment can be downloaded and installed using GitHub and pip.
```
Comment (zjowowen) [inline source/13_envs/evogym.rst:21]: The Evogym environment can be downloaded from GitHub and installed using pip.

File: source/13_envs/evogym.rst [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+
+   git clone --recurse-submodules https://github.com/EvolutionGym/evogym.git
+   cd evogym
+   pip install -r requirements.txt
+   python setup.py install
+
+Verify Installation
+--------------------
+
+After the installation is complete, you can verify that the installation was successful by running the following command on the command line:
```
Comment (zjowowen) [inline source/13_envs/evogym.rst:34]: After installation, you can verify whether if it is successful by running the following command:

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**